src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/controller/VisitController.kt

```diff
@@ -91,7 +91,7 @@ class VisitController(
   @PutMapping(VISIT_RESERVED_SLOT_CHANGE)
   @ResponseStatus(HttpStatus.OK)
   @Operation(
-    summary = "Change a reserved slot and associated details for a visit ",
+    summary = "Change a reserved slot and associated details for a visit (before booking)",
```

Comment:
@AledWEvans as discussed just now, reminder that this needs to cope with visit statuses that are `RESERVED` or `CHANGING`. (Probably worth tweaking the summary text there too so that's clear in the documentation)

Reply:
Yes will make change asap to change status during the VISIT_RESERVED_SLOT_CHANGE call if need be